aten/src/ATen/core/function_schema.h

```diff
@@ -126,6 +128,12 @@ struct FunctionSchema {
   const bool is_vararg_;
   const bool is_varret_;
 
+  // False if this schema was inferred for a custom operator. Custom ops infer
+  // a schema without alias annotations, so we need to:
+  //   1) know that the schema information is incomplete
+  //   2) treat the op conservatively w.r.t. aliasing and mutation
+  const bool has_alias_annotation_;
```

Comment:
nit: has_alias_annotation is misleading because unschematized ops don't have alias annotation either

Reply:
good point. I'll figure out a better name